broker-daemon/broker-rpc/admin-service/register.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Register the publicKey with the authorization service on the Relayer
+ *
+ * @param {GrpcUnaryMethod~request} request - request object
+ * @param {RelayerClient} request.relayer - grpc Client for interacting with the Relayer
+ * @param {Object} request.logger
+ * @param {Object} responses
+ * @param {function} responses.RegisterResponse - constructor for RegisterResponse messages
+ * @return {RegisterResponse}
+ */
+async function register ({ params, relayer, logger }, { RegisterResponse }) {
+  const { publicKey } = params
+
+  const { entityId } = await relayer.adminService.register({publicKey})
```

Comment:
I think we should probably persist this `entityId` id somewhere. Like in the `.sparkswap` directory